src/Api/Auth/Models/Request/Accounts/UpdateTdeOffboardingPasswordRequestModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Bit.Api.Auth.Models.Request.Accounts;
+
+public class UpdateTdeOffboardingPasswordRequestModel
+{
+    [Required]
+    [StringLength(300)]
+    public string NewMasterPasswordHash { get; set; }
+    [Required]
+    public string Key { get; set; }
```

Comment:
nit: I try to avoid naming things `Key` since we have a lot of keys 😁

Reply:
I generally agree, but in this case update-password and update-temp-password also use "key" so I'd rather have consistency between them and rename it only in tde offboarding.